boards/riscv/hpm6750evkmini/hpm6750evkmini_defconfig

```diff
@@ -0,0 +1,25 @@
+CONFIG_SOC_SERIES_HPMICRO=y
+CONFIG_SOC_HPM6750=y
+CONFIG_BOARD_HPM_6750EVKMINI=y
+CONFIG_XIP=n
+CONFIG_PLIC=y
+CONFIG_RISCV_MACHINE_TIMER=y
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_PINCTRL=y
+
+# Serial driver options
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+
+# GPIO driver options
+CONFIG_GPIO=y
+
+# HW DSP options
+#CONFIG_SOC_ANDES_V5_HWDSP=y
```

Comment:
Commented-out config option?

Reply:
Using dsp requires the user to provide a toolchain that supports the dsp command.

Reply:
Sure. Why isn't `CONFIG_SOC_ANDES_V5_HWDSP` defined in any Kconfig file? Or am I missing something?

Reply:
There was a problem with the first commit, and the second commit fixed the problem with the file `soc\riscv\riscv-privilege\hpmicro\Kconfig.soc`